crates/rpc/rpc/src/eth/api/pending_block.rs

```diff
@@ -43,13 +49,44 @@ impl PendingBlockEnv {
         let mut db = State::builder().with_database(Box::new(state)).with_bundle_update().build();
 
         let mut cumulative_gas_used = 0;
+        let mut sum_blob_gas_used = 0;
         let block_gas_limit: u64 = block_env.gas_limit.try_into().unwrap_or(u64::MAX);
         let base_fee = block_env.basefee.to::<u64>();
         let block_number = block_env.number.to::<u64>();
 
         let mut executed_txs = Vec::new();
         let mut best_txs = pool.best_transactions_with_base_fee(base_fee);
 
+        let (withdrawals, withdrawals_root) = match origin.clone() {
+            PendingBlockEnvOrigin::ActualPending(block) => {
+                (block.clone().withdrawals, block.withdrawals_root)
+            }
+            PendingBlockEnvOrigin::DerivedFromLatest(_) => (None, None),
+        };
```

Comment:
> f the pending block is an ActualPending then you know the parent_beacon_block_root, otherwise no, this is why I put None if pending block is not ActualPending

good point, didn't think of that